.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = http://github.com/genivi/meta-genivi-dev.git
 [submodule "meta-ivi"]
 	path = meta-ivi
-	url = http://git.yoctoproject.org/git/meta-ivi
+	url = https://github.com/GENIVI/meta-ivi.git
```

Comment:
why don't you use http or git protocol instead of https?
I know that https is also working.